src/engine/source/kvdb/src/kvdb.cpp

```diff
@@ -78,16 +78,14 @@ struct KVDB::Impl
         auto s = rocksdb::DB::ListColumnFamilies(kOptions.open, m_path, &cfNames);
         if (s.ok())
         {
-            for (auto name : cfNames)
+            for (const auto& name : cfNames)
```

Comment:
https://github.com/wazuh/wazuh/wiki/Coding-style-guide-(C-plus-plus)#performance

Use a for-range loop variable's as a reference type as many times as possible to avoid copying.